runbotics/runbotics-orchestrator-ui/src/views/process/AddProcess.tsx

```diff
@@ -34,6 +36,14 @@ const formData: IProcess = {
     definition: emptyBpmn,
 };
 
+enum ErrorType {
+    NAME_NOT_AVAILABLE,
```

Comment:
if you dont provide value for enum, by the default it has assigned 0, 1, 2 and so on.
lets put there string value, it can be the same as the key